WalletWasabi.Fluent/Views/AddWallet/Create/RecoveryWordsView.axaml

```diff
@@ -9,7 +9,7 @@
              x:DataType="create:RecoveryWordsViewModel"
              x:Class="WalletWasabi.Fluent.Views.AddWallet.Create.RecoveryWordsView">
   <c:ContentArea Title="{Binding Title}"
-                 Caption="Write down the following recovery words in the correct order and store them in a safe place. They are needed to recover this wallet."
+                 Caption="Write down your recovery words in the correct order and store them together in various safe places. They are needed to recover this wallet."
```

Comment:
Why are we suggesting multiple places? Every additional place you add exponentially weakens your security. So I think we shouldn't do that and the original phrasing was good. I personally find it hard enough to find a single secure place, let alone two or three.

I'd suggest to restore `in a safe place.` wording. Merge this PR. And if you feel strongly about multiple places, then open a new one to put it up for a discussion.
Everything else is good here IMO.